docs/reference/commandline/trust_key_generate.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "key-generate"
+description: "The key-generate command description and usage"
+keywords: "Key, notary, trust"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# trust key-generate
+
+```markdown
+Usage:  docker trust key-generate [OPTIONS] NAME 
```

Comment:
I don't think key generate has options this should be fixed in key_generate.go as well

Reply:
good catch.  I'll also have to update `sign` and `view` upstream.